other/upgrade_2-1_mysql.sql

```diff
@@ -1060,6 +1061,67 @@ ALTER TABLE {$db_prefix}members
 	DROP notify_announcements;
 ---#
 
+---# Upgrading auto notify setting
+---{
+$_GET['a'] = isset($_GET['a']) ? (int) $_GET['a'] : 0;
+$step_progress['name'] = 'Upgrading auto notify setting';
+$step_progress['current'] = $_GET['a'];
+
+$limit = 100000;
+$is_done = false;
+
+$request = $smcFunc['db_query']('', 'SELECT COUNT(*) FROM {db_prefix}themes WHERE variable = \'auto_notify\'');
```

Comment:
Missing the use of string replacement {string:auto_notify} .. array(auto_notify => 'auto_notify')
line break would be also needed in this case.

Reply:
Done